### PR TITLE
Update PBs for SetTopicPolicy

### PR DESCRIPTION
**Pull request by bbengfort**
This PR fixes SC-21883 which updates the SDK protocol buffers to reflect upcoming Ensign v0.11.0 changes including the `SetTopicPolicy` RPC.

I have made the following changes:

1. Updated the protocol buffers
2. Fixed tests broken by the changes in the protocol buffers
3. Updated `generate.sh` to do more error handling and fetch the protos path from an environment variable

This PR relates to https://github.com/rotationalio/ensign/pull/773 and https://github.com/rotationalio/go-ensign/pull/28

### TODOs and questions

Still to do:

- [ ] Add `set_topic_policy` method
- [ ] Add tests for setting the topic policy

### CHECKLIST

- [x] _Is the commit message formatted correctly?_
- [ ] _Do all of your functions and methods have docstrings?_
- [x] _Have you added/updated unit tests where appropriate?_
- [x] _Have you run the unit tests using `pytest`?_
- [x] _Is your code style correct (are you using PEP8, pyflakes)?_

### CODE REVIEW

File: generate.sh
Comment (bbengfort) [inline generate.sh:1]: My ensign repo is in my `$GOPATH` but pyensign is where I do my Python development and the packages are not related by the parent structure set, so I added a helper to use the `$ENSIGN_PROTOS` environment variable so I could specify a location without changing the behavior of the script.

I also enhanced the error handling a bit to make sure the script didn't explode with errors (like it did when I didn't have the path or when I didn't have grpcio-tools pip installed).

**Comment by bbengfort** [comment]
> Looks great so far, thanks for improving the protos build script. Did you need help with the API-level tests?

I've just added enumerations for the different policies, still need to add the OFFSET enumerations - mind taking another look so far?

**Comment by bbengfort** [comment]
@pdeziel ok, so I added one test - could you take a look and tell me if this is approximately what you were expecting? I think more tests are necessary but I kind of ran out of time here.

**Comment by bbengfort** [comment]
@pdeziel ok, I think I've sufficiently increased the test coverage here.

File: pyensign/enum.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
+from enum import Enum
+from pyensign.api.v1beta1 import topic_pb2
+
+
+class PBEnum(Enum):
```
Comment (pdeziel) [inline pyensign/enum.py:5]: Cool utility!

File: pyensign/enum.py [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    def parse(cls, name):
+        """
+        Parse an protocol buffer enumeration value from a string (case and whitespace
+        insensitive parsing). Raises a KeyError if the string is not in the enumeration.
+        """
+        name = name.strip().upper()
+        return cls[name]
+
+    @classmethod
+    def convert(cls, pb_val, name):
```
Comment (pdeziel) [inline pyensign/enum.py:21]: What do you think about `from_str` and `to_enum` for the names of these methods to make it more clear what they do?

**Review by pdeziel — APPROVED** [review]
Looks great, I appreciate the tests!

File: pyensign/enum.py (hunk 2 above)
Comment (bbengfort) [reply pyensign/enum.py:21]: Personally, `parse` and `convert` are more my preference but if you'd like to change them, please feel free.

**Pull request merged**